runtime/calamari/src/lib.rs

```diff
@@ -552,7 +552,7 @@ impl pallet_xcm::Config for Runtime {
 	type SendXcmOrigin = EnsureXcmOrigin<Origin, LocalOriginToLocation>;
 	type XcmRouter = XcmRouter;
 	type ExecuteXcmOrigin = EnsureXcmOrigin<Origin, LocalOriginToLocation>;
-	type XcmExecuteFilter = Everything;
+	type XcmExecuteFilter = Nothing;
```

Comment:
@stechu @Dengjianping 
The `nothing` trait implements `contains()` which always returns `false`
Then this check will not pass https://github.com/paritytech/polkadot/blob/master/xcm/pallet-xcm/src/lib.rs#L579
We have to implement this trait ourselves if we want to allow certain locations to pass this filter.